src/astarte-charts/react/ConnectedDevices/index.tsx

```diff
@@ -63,7 +63,7 @@ export const ConnectedDevicesChart = ({
       datasets: [
         {
           data: [disconnected.value, connected.value],
-          backgroundColor: ['#cc5b6d', '#5bcc6c'],
+          backgroundColor: ['#dc3545', '#33a344'],
```

Comment:
similarly to a previous comment, we could customize bootstrap `green` and `red` colors or defining reusable classes instead of hardcoding this here

Reply:
We are already customizing the `green` and `red` colors, however unfortunately we cannot specify CSS variables like `var(--bs-red)` as colors in ChartJS: https://www.chartjs.org/docs/latest/general/colors.html#color-formats